integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/nifi/v2/init/InitializeFeed.java

```diff
@@ -127,6 +135,13 @@
         .expressionLanguageSupported(true)
         .build();
 
+    public static final PropertyDescriptor CLEANUP_SERVICE = new PropertyDescriptor.Builder()
+        .name("Feed Cleanup Event Service")
+        .description("Service that manages the cleanup of feeds.")
+        .identifiesControllerService(CleanupEventService.class)
+        .required(true)
+        .build();
+
     Relationship REL_INITIALIZE = new Relationship.Builder()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "REL_INITIALIZE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)